OnlineShop/aspnet-core/src/OnlineShop.Domain/Bookss/Book.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Volo.Abp.Domain.Entities.Auditing;
+
+namespace OnlineShop.Bookss
+{
+    public class Book : AuditedAggregateRoot<Guid>
+    {
+        public int BookId { get; set; }
```

Comment:
Same issues as BookDto here, BookId is unneeded and Price should be a floating-point type